decentraland/unity-renderer
# feat: [BASE BRANCH] Improved Friend Requests

## What does this PR change?
This is a part of the implementation of the new Friend Requests feature.

Right now we have a feature flag called `new_friend_requests` that is deactivated in `.org` and ensures that none of the new implemented flows are going to be executed, so only the old ones should be actived.

Te idea is, once we are sure that all the old "Friend Request" flows are working fine, to merge this PR to `dev` and continue the implementation of the new feature in other PRs.

## How to test the changes?
1. Go to: https://play.decentraland.org/?renderer-branch=feat/improved-friend-requests
2. Check that all the old behaviors related to Friend Requests keep working as usual
3. Try sending, cancelling, rejecting, accepting requests from different sources: user context menu (by clicking on a user name in the chat), passport card (by clicking on an avatar in the world), the friend requests tab (clicking the friends button in the taskbar)

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
